software/ompi/v1.0/index.php

```diff
@@ -1,70 +1,28 @@
 <?php
 $topdir = "../../..";
 
-include_once("$topdir/software/ompi/v1.0/version.inc");
-$title = "Open MPI: Version $ver_v1_0";
+include_once("version.inc");
+$title = "Open MPI: Version $release_series";
 
 include_once("$topdir/software/ompi/nav.inc");
 include_once("$topdir/includes/header.inc");
 include_once("$topdir/includes/downloads.inc");
 
-$md5 = read_checksums("downloads/md5sums.txt");
-$sha1 = read_checksums("downloads/sha1sums.txt");
+$project = "Open MPI";
+$list_name = "announce";
+$prev_describe = "the v$release_series download page";
 
-#############################################################################
-
-$t = new downloadTable("./downloads", "./downloads",
-		       dirname($_SERVER["PHP_SELF"]));
-
-$src = "v$ver_v1_0 (stable) source code (<a href=\"srpm.php\">SRPM notes</a>)";
-$name = "openmpi-$ver_v1_0.tar";
-$t->addFile($src, "$name.bz2", $md5["$name.bz2"], $sha1["$name.bz2"]);
-$t->addFile($src, "$name.gz", $md5["$name.gz"], $sha1["$name.gz"]);
-$name = "openmpi-$ver_v1_0-1.src.rpm";
-$t->addFile($src, $name, $md5[$name], $sha1[$name]);
-
-# No release candidates at the moment
-#$s = "v1.0.2 (pending) alpha";
-#$fp = fopen("downloads/latest_snapshot.txt", "r");
-#$v = fgets($fp);
-#fclose($fp);
-#$v = substr($v, 0, strlen($v) - 1);
-#
-#$t->addFile($s, "openmpi-$v.tar.bz2",
-#            $md5["openmpi-$v.tar.bz2"],
-#            $sha1["openmpi-$v.tar.bz2"]);
-#$t->addFile($s, "openmpi-$v.tar.gz",
-#            $md5["openmpi-$v.tar.gz"],
-#            $sha1["openmpi-$v.tar.gz"]);
-
-$src = "v1.0.1 (older release) source code";
-$name = "openmpi-1.0.1.tar";
-$t->addFile($src, "$name.bz2", $md5["$name.bz2"], $sha1["$name.bz2"]);
-$t->addFile($src, "$name.gz", $md5["$name.gz"], $sha1["$name.gz"]);
-
-$src = "v1.0 (older release) source code";
-$name = "openmpi-1.0.tar";
-$t->addFile($src, "$name.bz2", $md5["$name.bz2"], $sha1["$name.bz2"]);
-$t->addFile($src, "$name.gz", $md5["$name.gz"], $sha1["$name.gz"]);
-
-#############################################################################
-#
-# Main display part of the page
-#
-#############################################################################
-
-$prev_describe = "the v$ver_v1_2 download page";
 include_once("$topdir/includes/subscribe-announce.inc");
 ?>
 
 <p><hr>
 
 <p><strong><font color=red>PLEASE NOTE:</font></strong> The 1.0 series
 of Open MPI is effectively dead.  There was work beyond the last
-stable release (v<?php print($ver_v1_0); ?>) in the Subversion repository
+stable release (v1.0.2) in the Subversion repository
 that was never released as a stable package (but is available from the
 <code>/branches/v1.0</code> tree in the SVN repository and the <a
-href="<?php print($topdir); ?>/nightly/v1.0/">v1.0 snapshot
+href="../../../nightly/v1.0/">v1.0 snapshot
```

Comment:
Just curious why you removed the `<?php print...` blocks from these two sections and instead put the hard-coded strings.  I only ask because you didn't do this elsewhere.

Reply:
v1.0 was where I did most of the initial work.  I think the `$topdir` was part of a failed attempt at reducing the number of places we cared about where the top was, trying to reduce duplicate code even further.

The variable -> 1.0.2 hardcode a couple lines above was laziness combined with a knowledge that we were never going to ship 1.0.3.